trillian_log_api.proto

```diff
@@ -82,9 +82,10 @@ service TrillianLog {
   }
 
   // Returns the latest signed log root for a given tree. Corresponds to the
-  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  The server will
-  // return InvalidArgument if first_tree_size is greater than the
-  // LatestSignedLogRoot available to that server.
+  // ReadOnlyLogTreeTX.LatestSignedLogRoot storage interface.  If
+  // first_tree_size > latest SignedLogRoot.tree_size, the server will return
+  // gRPC `codes.Unavailable` with the latest SignedLogRoot attached in the
+  // gRPC status.Details().
```

Comment:
No such thing as `status.Details()`, only `status.Details` or `status.GetDetails()` (former preferred).

Reply:
I'd say even more than preferred, because this is a protobuf file, not a Go file, so it shouldn't have "Goisms", what you're really referring to is `google.rpc.Status.details`:

https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto#L91